vyper/evm/opcodes.py

```diff
@@ -75,6 +75,8 @@
     "CHAINID": (0x46, 0, 1, 2),
     "SELFBALANCE": (0x47, 0, 1, 5),
     "BASEFEE": (0x48, 0, 1, 2),
+    "BLOBHASH": (0x49, 1, 1, 3),
```

Comment:
should we also support this? i guess it should probably be supported similarly to blockhash

Reply:
Yes, I think we should. I have done some work on it locally but I am still figuring out how to test it. How about moving this to a separate PR so it does not block `block.blobbasefee`?